I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 12, in script call
    call scene_1
  File "game/script.rpy", line 19, in script
    show akari at from_offscreen
  File "game/script.rpy", line 19, in <module>
    show akari at from_offscreen
NameError: name 'from_offscreen' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 12, in script call
    call scene_1
  File "game/script.rpy", line 19, in script
    show akari at from_offscreen
  File "C:\Users\crank\Documents\renpy-8.3.4-sdk\renpy\ast.py", line 1117, in execute
    show_imspec(self.imspec, atl=getattr(self, "atl", None))
  File "C:\Users\crank\Documents\renpy-8.3.4-sdk\renpy\ast.py", line 1076, in show_imspec
    at_list = [ renpy.python.py_eval(i) for i in at_list ]
  File "C:\Users\crank\Documents\renpy-8.3.4-sdk\renpy\ast.py", line 1076, in <listcomp>
    at_list = [ renpy.python.py_eval(i) for i in at_list ]
  File "C:\Users\crank\Documents\renpy-8.3.4-sdk\renpy\python.py", line 1209, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Users\crank\Documents\renpy-8.3.4-sdk\renpy\python.py", line 1202, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 19, in <module>
    show akari at from_offscreen
NameError: name 'from_offscreen' is not defined

Windows-10-10.0.22631 AMD64
Ren'Py 8.3.4.24120703
What We Leave Behind 1.0
Sun Jan 19 08:44:57 2025
